src/librustdoc/lib.rs

```diff
@@ -5,6 +5,7 @@
 #![feature(rustc_private)]
 #![feature(assert_matches)]
 #![feature(box_patterns)]
+#![feature(debug_closure_helpers)]
```

Comment:
Minor: This change accidentally made it into an unrelated commit, namely https://github.com/rust-lang/rust/pull/135494/commits/b32e04042c0258694da3a8a3c6b382734d0e4b50. No need to fix unless you'd like to ofc.